### ISSUE TITLE
different label pad token id between layoutxlm and ernie layout

### ISSUE THREAD

**Issue by vanpersie32** [NormXU/ERNIE-Layout-Pytorch#8]
Hi, I find that the label_pad_token id is different between layoutxlm and ernie layout in token classification. In layoutxlm, the label_pad_token id is -100 and  will be ignored by pytorch loss functions [url](https://github.com/huggingface/transformers/blob/e5dcceb82ca2c1e0d23a57f4856940a36c810a5b/src/transformers/data/data_collator.py#L290). However the label_pad_token id is 0 in ernie layout and will be calculated loss.  [url2](https://github.com/NormXU/ERNIE-Layout-Pytorch/blob/main/examples/test_ernie_token_cls.py#L19-L27)

**Comment by NormXU** [comment]
Thanks for pointing out the bug. I've fixed it.
